cli/backup.go

```diff
@@ -44,11 +45,17 @@ func runBackup(cn *pbm.PBM, b *backupOpts, outf outFormat) (fmt.Stringer, error)
 		return nil, errors.Wrap(err, "get remote-store")
 	}
 
+	var level *int
```

Comment:
If someone has a suggestion to avoid this dirty hack, I'm open to suggestion 😀 
But I couldn't find any other way with the Kingpin library to support an optional integer argument which has to be `nil` when not provided.

Reply:
I propose using `-1` as the default. kinping's flags have Default() functions so we're safe here. 
The only potential pitfall is the straightforward usage of [BackupCMD](https://github.com/percona/percona-backup-mongodb/pull/619/files#diff-71edcf47fb2c9f5a756a0b6aa78b3a5609368d7cf52fdcf64acf8660ab0eeec2R158) But we can add some comments there so a user would be aware of default bechaviour. 

BTW, I took a look and pgzip's DefaultCompression is -1 as well

Reply:
There were 2 reasons why I didn't pick `-1` as equivalent of default instead of `nil`:
1. There could be compression methods that support/use `-1` as a valid, non-default value. Actually, I see that the Zstandard documentation mentions such levels: https://facebook.github.io/zstd/zstd_manual.html.
2. It doesn't play well with the `omitempty` functionality in Go, where a non specified integer gets `0` assigned.

As in:
```golang
type BackupCmd struct {
	Name             string          `bson:"name"`
	Compression      CompressionType `bson:"compression"`
	CompressionLevel int             `bson:"level,omitempty"`
}
```

On the other hand, having `CompressionLevel *int` as I proposed, means being able to differentiate between `0` as "no value passed" and `0` as "level 0 specifically".